fix: no line breaks for placholder (#8949)
shopware/platform: changelog/_unreleased/2025-04-30-fix-no-new-line-for-placeholder.md

---
title: fix no new line for placeholder
issue: #7476
---
# Administration

> * Changed `src/Administration/Resources/app/administration/src/app/component/form/select/entity/sw-entity-single-select/sw-entity-single-select.scss`. Placeholder will not break lines and will not overflow

Reviewer: ```suggestion
* Changed `src/app/component/form/select/entity/sw-entity-single-select/sw-entity-single-select.scss` to not have a line break on placeholder overflow.
```
